Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -368,6 +368,18 @@ public bool OnlyUseDefaultSpriteName
             return !string.IsNullOrEmpty(getSpriteNameAction);
         }
     }
+
+    /// <summary>
+    /// Whether the furniture has power or not.
+    /// </summary>
+    /// <returns>True if the furniture has power.</returns>
+    public bool HasPower
+    {
+        get
+        {
+            return PowerConnection == null || World.Current.PowerNetwork.HasPower(PowerConnection);
```

Comment:
Please change the logic so `IsOperating `does not depend on `HasPower `for items that don't require power. That way items accurately refect their powered state